mailgun/listserv_client.py

```diff
@@ -221,6 +221,14 @@ def send_mail(self, from_address, reply_to_address, to_address, cc_address,
             'html': html
         }
 
+        # we accept a single address or a list of addresses in to_address.
+        # if it's a list, add in recipient_variables to make sure mailgun
+        # doesn't include the whole list in the to: field, per
+        #   https://documentation.mailgun.com/user_manual.html#batch-sending
+        if not isinstance(to_address, basestring):
```

Comment:
So the end result is we can *almost* replicate the original to/cc fields.  There's no way to prevent mailgun from adding the individual recipient to the To field, but the rest of the to/cc should look the same as it was send.